src/notebook/widget.ts

```diff
@@ -337,6 +337,25 @@ class NotebookWidget extends Widget {
 }
 
 
+function createElements(tag: string, attr?: any, children?: HTMLElement[] | string): HTMLElement {
```

Comment:
The whole point of the convenience function is to make the structure of the code resemble the structure of the html, so it is easy to quickly grasp the sort of html coming out. For that reason, I think the children should definitely be an argument to the function. I agree that since we don't use very many attributes, it's helpful to have the typechecking on those. And yep, I agree that the function should be `createElement`.